src/test/java/test/com/epam/training/geometry/data/FileDataReaderTest.java

```diff
@@ -0,0 +1,42 @@
+package test.com.epam.training.geometry.data;
+
+
+import com.epam.training.geomety.data.DataException;
+import com.epam.training.geomety.data.FileDataReader;
+import org.junit.Assert;
+import org.junit.Test;
+
+import java.util.List;
+
+public class FileDataReaderTest {
+
+    private final FileDataReader reader = new FileDataReader();
+    private static final String BASE_RESOURCE_DIRECTORY = "./src/test/resources/dataTest/";
+
+    @Test(expected = DataException.class)
+    public void testReadLinesShouldThrowDataExceptionWhenFileDoesntExist() throws DataException {
+        String filename = BASE_RESOURCE_DIRECTORY + "doesntExist.txt";
```

Comment:
define constants:
INPUT_FILE
INVALID_FILE
don not concatinate at all and here as well, just full path constants